PowerVCF.psm1

```diff
@@ -121,15 +121,17 @@ public static class Placeholder {
             $response = Invoke-RestMethod -Method POST -Uri $uri -Headers $headers -Body $body -SkipCertificateCheck # PS Core has -SkipCertificateCheck implemented
             $Global:accessToken = $response.accessToken
             $Global:refreshToken = $response.refreshToken.id
-        } else {
+        }
+        else {
             $response = Invoke-RestMethod -Method POST -Uri $uri -Headers $headers -Body $body
```

Comment:
It looks like you're editor config is automatically changing these in many places?